tools/cell_census_builder/tests/conftest.py

```diff
@@ -60,20 +63,20 @@ def get_h5ad(organism: Organism) -> anndata.AnnData:
     obs = obs_dataframe
 
     # Create vars
-    feature_name = pd.Series(data=[f"{organism.name}_{g}" for g in ["a", "b", "c", "d"]])
+    feature_id = pd.Series(data=[f"{organism.name}_{g}" for g in gene_ids])
     var_dataframe = pd.DataFrame(
         data={
             "feature_biotype": "gene",
             "feature_is_filtered": False,
             "feature_name": "ERCC-00002 (spike-in control)",
             "feature_reference": organism.organism_ontology_term_id,
         },
-        index=feature_name,
+        index=feature_id,
     )
     var = var_dataframe
 
     # Create embeddings
-    random_embedding = np.random.rand(4, 4)
+    random_embedding = np.random.rand(cells, genes)
```

Comment:
tiny nit: this is using an obsolete numpy API.  New code would ideally use the Generator API.

(optional to fix)